### PR TITLE
 HLT menu for XeXe MC production (92X)

### PR DESCRIPTION
**Pull request by silviodonato**
Adding HLT XeXe frozen menu to be used in the XeXe MC production.
The menu works with L1Menu_Collisions2017_v3_m6_xml.

94X PR: #21315
Since this menu will be only used for the XeXe MC production, there is no 100X PR.

### CODE REVIEW

**Comment by silviodonato** [comment]
please test

File: HLTrigger/Configuration/test/getFrozenHLT.sh [hunk 1]
```diff
@@ -1,13 +1,15 @@
 #! /bin/bash
 
 # ConfDB configurations to use
-TABLES="Fake Fake1 Fake2 2e34v22 2e34v31 2e34v40"
+TABLES="XeXe Fake"
+#Fake Fake1 Fake2 2e34v22 2e34v31 2e34v40
```
Comment (Martin-Grunewald) [inline HLTrigger/Configuration/test/getFrozenHLT.sh]: Can you please restore the other frozen menus?

**Comment by silviodonato** [comment]
please test

**Comment by silviodonato** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**